ludwig/trainers/trainer_llm.py

```diff
@@ -370,7 +365,7 @@ def evaluation(
         progress_tracker: ProgressTracker,
     ):
         predictor = Predictor(
-            self.model, batch_size=batch_size, distributed=self.distributed, report_tqdm_to_ray=self.report_tqdm_to_ray
+            self.model, batch_size=1, distributed=self.distributed, report_tqdm_to_ray=self.report_tqdm_to_ray
         )
         metrics, _ = predictor.batch_evaluation(dataset, collect_predictions=False, dataset_name=dataset_name)
```

Comment:
Why are we using a batch size of 1 here? Is there a way to grab it from the schema potentially?

Reply:
good catch– leftover from debugging. fixed.